composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "adldap2/adldap2-laravel",
+    "name": "minkbear/adldap2-laravel",
```

Comment:
sorry I do not know when my PR will be merge and I have a deadline to use my modified package by using composer(https://packagist.org/) to be sure that I can release for my client.

I welcome to send new PR. what do you think?